python/cudf/_gdf.py

```diff
@@ -344,8 +344,8 @@ def libgdf_join(col_lhs, col_rhs, on, how, method='sort'):
     return res, valids
 
 
-def apply_prefixsum(col_inp, col_out, inclusive):
-    libgdf.gdf_prefixsum(col_inp, col_out, inclusive)
+def apply_scan(col_inp, col_out, op, inclusive):
+    libgdf.gdf_scan(col_inp, col_out, op, inclusive)
```

Comment:
We area actualy in the process of moving to Cython instead of cffi. So you can remove this function and make a new file that contains all reduction/scan based functions in `python/bindings`